charts/kubewarden-controller/Chart.yaml

```diff
@@ -21,9 +21,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
```

Comment:
(open thread here randomly)

These are the changes between GA and this RC:
https://github.com/kubewarden/helm-charts/compare/kubewarden-defaults-1.7.3...updatecli_9cdd3756d921d5ada8a9fcc0ef40ad745a44079a53c2b4c0bdaf0e307eceb4b9